tests/conftest.py

```diff
@@ -132,11 +133,15 @@ def confluent_admin_client(test_config: Config) -> AdminClient:
 
 
 @pytest.fixture()
-def producer(topic_object: Topic, cluster: Cluster):
-    # Send messages synchronously so we can be sure offset has been commited in tests.
-    yield Producer(
-        cluster.pykafka_client.cluster, topic_object._pykafka_topic, sync=True
-    )
+def producer(test_config: Config):
+    producer_config = test_config.create_confluent_config()
+    yield Producer(producer_config)
+
+@pytest.fixture()
+def avro_producer(test_config: Config):
+    producer_config = test_config.create_confluent_config()
+    producer_config.update({"schema.registry.url": Config().schema_registry})
```

Comment:
Could it be that you want `test_config.schema_registry` instead of  `Config().schema_registry` ?

Reply:
That's just in here because i merged in master :D